recipes/jetty.rb

```diff
@@ -93,17 +93,29 @@
   end
 end
 
+solr_config_path =  case node['platform_family']
+                    when 'rhel'
+                      '/etc/sysconfig/solr'
+                    when 'debian'
+                      '/etc/default/solr'
+                    end
+
 template 'solr_config' do
   source 'solr.conf.erb'
   owner node['solrcloud']['user']
   group node['solrcloud']['group']
   mode 0744
-  case node['platform_family']
-  when 'rhel'
-    path '/etc/sysconfig/solr'
-  when 'debian'
-    path '/etc/default/solr'
-  end
+  path solr_config_path
+  only_if { node['solrcloud']['major_version'] <= 5 && node['solrcloud']['minor_version'] < 2 }
+end
+
+template 'solr_config' do
+  source 'v5.2.x/solr.conf.erb'
+  owner node['solrcloud']['user']
+  group node['solrcloud']['group']
+  mode 0744
+  path solr_config_path
+  only_if { node['solrcloud']['version'] > '5.2' }
 end
 
 template '/etc/init.d/solr' do
```

Comment:
how about calculating source in default attributes? like:

```ruby
if node['solrcloud']['version'] >= '5.2'
  default['solrcloud']['init_source'] = 'v5.2.x/solr.conf.erb'
else
  default['solrcloud']['init_source'] = "#{node['platform_family']}.solr.init.erb"
end
```

i hope template 'v5.2.x/solr.conf.erb' works for both `debian` and `rhel` platform families. will test it for `rhel` anyway.